libs/injection/bbhnet/injection/nonspin_BBH.prior

```diff
@@ -0,0 +1,15 @@
+mass_1 = Uniform(name='mass_1', minimum=5, maximum=100, unit='$M_{\odot}$', boundary=None)
+mass_2 = Uniform(name='mass_2', minimum=5, maximum=100, unit='$M_{\odot}$', boundary=None)
+mass_ratio = Constraint(name='mass_ratio', minimum=0.2, maximum=5.0)
+luminosity_distance = bilby.gw.prior.UniformSourceFrame(name='luminosity_distance', minimum=100, maximum=200, unit='Mpc', boundary=None)
+dec = Cosine(name='dec')
+ra = Uniform(name='ra', minimum=0, maximum=2 * np.pi, boundary='periodic')
+theta_jn = 0
+psi = 0
+phase = 0
+a_1 = 0
+a_2 = 0
+tilt_1 = 0
+tilt_2 = 0
+phi_12 = 0
+phi_jl = 0
```

Comment:
I'm not sure this prior file makes sense to include in the library, since different applications will want to use their own sets of priors. This will definitely make sense to include in e.g. a project for making background and signal distributions on the same stretch of data

Reply:
Good point, I've removed this